README.adoc

```diff
@@ -0,0 +1,63 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
+
+This repository contains add-ons contributed by the Minishift community.
+
+We hope that by sharing these add-ons, folks in the community will be able to make the most out of their Minishift environment.
+Community add-ons are tested and supported by the community.
+
+[[introduction-addons]]
+== Introduction to add-ons
+
+link:https://docs.openshift.org/latest/minishift/using/addons.html[Minishift add-ons] are extensions, or plug-ins, that customize the default OpenShift cluster configuration and behavior.
+For example, you can use an add-on to allow authenticated users to run images without a pre-allocated UID, or to run commands on the Docker daemon inside the Minishift VM.
+
+Minishift includes several link:https://docs.openshift.org/latest/minishift/using/addons.html#default-addons[default add-ons], which are stored in *_$MINISHIFT_HOME/addons_*.
+These add-ons are tested and supported by the Minishift core team, although due to the experimental nature of the add-ons mechanism, backwards-compatibility is not guaranteed.
+You can also write your own link:https://docs.openshift.org/latest/minishift/using/addons.html#custom-addons[custom add-ons] for any of the link:https://docs.openshift.org/latest/minishift/using/addons.html#addon-commands[supported commands].
+
+[[share-addons-with-community]]
+== Share your add-ons with the Minishift community
+
+You are invited to share your add-ons with the Minishift community by submitting them to this repository.
+To submit your add-on, follow the link:CONTRIBUTING.adoc[contribution guidelines] to open a new issue and submit a pull request.
+
+Community add-ons are not officially supported by the Minishift core team, although the core developers do their best to help out when reviewing and merging new add-ons to this repository.
+The community is encouraged to help contributors by testing and improving these add-ons.
+
+[[download-use-community-addons]]
+== Download and use community add-ons
+
+To use any of the add-ons from this repository in your Minishift environment:
+
+. Clone the repository to your local machine.
+
+. Install and enable the add-on that you want to use by running the following command:
```

Comment:
@hferentschik see @LalatenduMohanty 's comment, the command syntax you asked me to write is to install and enable, not apply. I recommend not diving too much into details here because there is a whole section of the minishift docs about applying, enabling, and disabling add-ons. We have a link from here to that section, so I would advise writing only the minimum quickstart info here.